lib/uknetdev/Config.uk

```diff
@@ -31,6 +31,24 @@ config LIBUKNETDEV_DISPATCHERTHREADS
 		allocated for each configured receive queue.
 		libuksched is required for this option.
 
+config LIBUKNETDEV_EINFO_LIBPARAM
```

Comment:
Should we add a `default y` here, since currently we run all applications using kernel params, and this will need to be  enabled for that to work?

Reply:
No, because Unikraft options should build a minimal kernel as default with all features off. The idea is that application configurations (application's `Config.uk` or `Kraftfile`) should take care of enabling needed options. We could put an `imply` option to `app-elfloader`, for example.